### PR TITLE
Enable support for verify=False set in requests library.

### PR DESCRIPTION
**Pull request by fuzzyrichie**
See GitHub issue https://github.com/cdent/wsgi-intercept/issues/59

### CODE REVIEW

**Comment by cdent** [comment]
This looks great, thanks very much. I'll wait for CI to say it's good and then will happily merge this. Thanks.

**Comment by fuzzyrichie** [comment]
Sounds good - thanks for the help @cdent! I just upped the version to 1.9.1 as well - let me know if that is OK with you.

**Comment by fuzzyrichie** [comment]
Hmm, odd that there are that many failures. Will tackle those now and get back to you when I get it in a good state.

**Comment by cdent** [comment]
Initially I thought the problem might be the version of urllib3 that is being installed on travis. I had 1.25.6 locally but upgrading to 1.25.7 they still work.

Another thing I thought of was travis talking to the network being an issue, but it does that for other tests.

There is a mechanism for preventing network tests, but it isn't turned on for travis. See `skipnetwork` and `WSGI_INTERCEPT_SKIP_NETWORK` in the code for a bit more on that but that may be a red herring.

**Comment by fuzzyrichie** [comment]
@cdent - the issue was that the internals for the http module are slightly different between Py2/3.5/3.6/3.7. Which is unfortunate, but hopefully these fixes make it work properly.

File: wsgi_intercept/__init__.py [hunk 1]
```diff
@@ -640,7 +658,8 @@ def connect(self):
                             else:
                                 self._check_hostname = self.check_hostname
                 except (ImportError, AttributeError):
-                    pass
+                    import traceback
+                    traceback.print_exc()
```
Comment (fuzzyrichie) [inline wsgi_intercept/__init__.py]: Unsure if you want me to remove this (was used for debugging) or leave it there?

Comment (cdent) [reply wsgi_intercept/__init__.py]: It's fine to leave it there.

**Comment by cdent** [comment]
Thanks for doing this. Sorry it was a bit more of a hassle than it initially seemed. Much of the wsgi-intercept code is very old (and predates my involvement) and the whole thing is a series of hacks...

I'll get it merged and a new release done some time today.

**Comment by cdent** [comment]
I'm going to make some docs and testing tweaks before doing a release.

**Comment by cdent** [comment]
1.9.1 has been released with this

**Pull request merged**